.github/linters/lychee.toml

```diff
@@ -0,0 +1,13 @@
+exclude = ["docs/api/javadoc/sql/org/apache/sedona/sql/utils/index.html$",
+    "docs/api/scaladoc/spark/org/apache/sedona/sql/utils/index.html$",
+    "docs/javadoc/spark",
+    "docs/scaladoc/spark/org/apache/sedona/sql/utils/index.html$",
+    "spark://localhost:.*",
+    "https://issues.apache.org/jira.*",
+    "https://keyserver.pgp.com/.*",
+    "http://localhost:.*",
+    "https://whimsy.apache.org/officers/acreq.*",
+    "https://whimsy.apache.org/roster/pmc/sedona.*",
+    "https://www.datasyslab.net/",
```

Comment:
Hey @jiayuasu this `datasyslab` link is broken

https://github.com/apache/sedona/blob/181ad31be351ffa459b5d57f078bd034adebb9f8/docs/community/publication.md?plain=1#L3

Reply:
@jbampton Can you replace it with this one: https://datasystemslab.engineering.asu.edu/